docs/modules/ROOT/pages/how-tos/testing/build/index.adoc

```diff
@@ -1 +1,27 @@
-= Build-time tests
\ No newline at end of file
+= Build-time tests
+
+This document covers the build-time tests (formerly known as "sanity tests") that {ProductName} runs as part of its component build pipeline. These build-time tests automatically check all application images to ensure that they're up-to-date, correctly formatted, and protected from security vulnerabilities.
```

Comment:
These tests are all enabled by default by configuration of the default parameter:
```
    - description: Skip checks against built image
      name: skip-checks
      type: string
      default: "false"
```
Do we want to inform users how to disable them by setting that parameter?

Reply:
@arewm How about this?

`You can disable the default build-time tests by setting the `skip-checks` parameter to `true`

[source,yaml]
----
- description: Skip checks against built image
  name: skip-checks
  type: string
  default: "false"
----`

Reply:
Not sure if I get this, but based on my understanding I believe we can add a page like "Updating build time tests" and add a link here. WDYT?

Reply:
It wouldn't be specific to build time tests as it is just about customizing the build pipeline (which potentially includes these build-time tests).

Reply:
As discussed this in chat, I'll update this with content from other PR and will ask for your review. 
Marking this resolved.